components/sup/src/manager/service/mod.rs

```diff
@@ -367,6 +373,19 @@ impl Service {
         spec
     }
 
+    fn assert_bindings_present(&self, census_list: &CensusList) -> bool {
```

Comment:
is `assert` appropriate here since this function does not error if the assertion is false? I'm not certain this is a hard convention in rust but seems like it may be.